integration-testing/config-local-host.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+set -e
```

Comment:
maybe: `set -ex` and we can see which commands are running
but don't do that if it's printing "secrets" we don't want anyone to know